### ISSUE TITLE
extract text from anywhere? How about the camera?

### ISSUE THREAD

**Issue by igalic** [TenderOwl/Frog#45]
I'd like to request a feature for grabbing a shot from the camera, as well to extract text from.
Especially with QR Codes, this seems like a useful thing, given how ephemeral they often are

**Comment by amka** [comment]
Hey, it is a good idea! I have to make some reasearch about that, but it definitely deserve a try :)Отправлено с устройства Galaxy
-------- Исходное сообщение --------От: Mina Galić ***@***.***> Дата: 28.04.2022  14:39  (GMT+03:00) Кому: TenderOwl/Frog ***@***.***> Копия: Subscribed ***@***.***> Тема: [TenderOwl/Frog] extract text from anywhere? How about the camera? (Issue #45) 
I'd like to request a feature for grabbing a shot from the camera, as well to extract text from.
Especially with QR Codes, this seems like a useful thing, given how ephemeral they often are

**Comment by jakob1379** [comment]
Can't imagine it would be too hard grabbing the stream and applying it to each 10th or 20th image

**Comment by amka** [comment]
You're right. Unfortunately, the problems come from Feb 24, and for now, I have no permanent home and time to get things done. There are a lot of ideas about what to add to the Frog, but this year went crazy and I didn't handle it yet.

**Comment by jakob1379** [comment]
No stress! I might make a mockup for reference :D

**Comment by amka** [comment]
Better late than never. At the current stage, I have a prototype that allows you to connect a camera and receive stream and frames, however, so far I have not been able to set up the GTS to give at least 30 frames per second. Because of this, frames with text are blurry and it is not possible to recognize such text. But I'm working on it.

**Comment by gregorni** [comment]
How about doing it so that instead of a stream, you just let the user take a photo, then pass that to the app. That should be enough for most use cases, right?

**Comment by gae-rafatjah** [comment]
Crazy idea but maybe open the camera app! Love how great this tool is. Simple and solid. Adding a ton of features makes it unreliable. Thanks for all the time and effort.
